src/app/generate/generate.go

```diff
@@ -30,13 +30,13 @@ const (
 const emptyChangelogOutput = "empty-changelog"
 
 // ErrNoSources is returned if Generate is invoked without any source enabled.
-var ErrNoSources = errors.New("cannot generate changelog without at least one source enabled")
+var ErrNoSources = errors.New("cannot generate changelog yaml without at least one source enabled")
 
-// Cmd is the cli.Command object for the generate command.
+// Cmd is the cli.Command object for the generate-yaml command.
 //
 //nolint:gochecknoglobals // We could overengineer this to avoid the global command but I don't think it's worth it.
 var Cmd = &cli.Command{
-	Name:  "generate",
+	Name:  "generate-yaml",
 	Usage: "Builds a changelog.yaml file from multiple sources",
```

Comment:
wouldn't make sense to use the same comment from the action here? i think the improved action description is way more clear.
Same for the rest of commnads